lib/basket.rb

```diff
@@ -0,0 +1,38 @@
+require 'send_sms'
```

Comment:
required this file, but it doesn't exsist. This throws the error 'LoadError (cannot load such file -- send_sms)' when trying to require basket in rb.